so far with 6.6.26
rpm_msg_ram: 	<0x0 0x00060000 0x0 0x6000>
bootloader:	<0x0 0x4a100000 0x0 0x400000>
sbl:		<0x0 0x4a500000 0x0 0x100000>
tz: 		<0x0 0x4a600000 0x0 0x400000> 
smem_region:	<0x0 0x4aa00000 0x0 0x100000>
q6_region:	<0x0 0x4ab00000 0x0 0x5500000>

it doesn't have
nss_region:	<0x0 0x40000000 0x0 0x01000000>
q6_etr_region:	<0x0 0x50000000 0x0 0x100000>
m3_dump_region:	<0x0 0x50100000 0x0 0x100000>
include 'em
that nss seems needed for stability, which is weird.

--- a/arch/arm64/boot/dts/qcom/ipq6018.dtsi	2024-03-28 14:14:27.431424230 +0800
+++ b/arch/arm64/boot/dts/qcom/ipq6018.dtsi	2024-03-28 14:14:01.366582092 +0800
@@ -194,6 +194,11 @@
 			no-map;
 		};
 
+		nss_region: nss_region@40000000 {
+			reg = <0x0 0x40000000 0x0 0x01000000>;
+			no-map;
+		};
+
 		bootloader@4a100000 {
 			reg = <0x0 0x4a100000 0x0 0x400000>;
 			no-map;
@@ -218,6 +223,16 @@
 			reg = <0x0 0x4ab00000 0x0 0x5500000>;
 			no-map;
 		};
+
+		q6_etr_region: q6_etr_region@50000000 {
+			reg = <0x0 0x50000000 0x0 0x100000>;
+			no-map;
+		};
+
+		m3_dump_region: m3_dump_region@50100000 {
+			reg = <0x0 0x50100000 0x0 0x100000>;
+			no-map;
+		};
 	};
 
 	smem {
